presto-tests/src/test/java/io/prestosql/tests/TestInformationSchemaConnector.java

```diff
@@ -0,0 +1,143 @@
+        assertQuery("SELECT count(*) FROM tpch.information_schema.tables", "VALUES (80)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.columns", "VALUES (585)");
+        assertQuery("SELECT * FROM tpch.information_schema.schemata ORDER BY 1 DESC, 2 DESC LIMIT 1", "VALUES ('tpch', 'tiny')");
+        assertQuery("SELECT * FROM tpch.information_schema.tables ORDER BY 1 DESC, 2 DESC, 3 DESC, 4 DESC LIMIT 1", "VALUES ('tpch', 'tiny', 'supplier', 'BASE TABLE')");
+        assertQuery("SELECT * FROM tpch.information_schema.columns ORDER BY 1 DESC, 2 DESC, 3 DESC, 4 DESC LIMIT 1", "VALUES ('tpch', 'tiny', 'supplier', 'suppkey', 1, NULL, 'YES', 'bigint', NULL, NULL)");
+    }
+
+    @Test
+    public void testSchemaNamePredicate()
+    {
+        assertQuery("SELECT count(*) FROM tpch.information_schema.schemata WHERE schema_name = 'sf1'", "VALUES (1)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.tables WHERE table_schema = 'sf1'", "VALUES (8)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.columns WHERE table_schema = 'sf1'", "VALUES (61)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.columns WHERE table_schema = 'information_schema'", "VALUES (36)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.columns WHERE table_schema > 'sf100'", "VALUES (427)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.columns WHERE table_schema != 'sf100'", "VALUES (524)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.columns WHERE table_schema LIKE 'sf100'", "VALUES (61)");
+    }
+
+    @Test
+    public void testTableNamePredicate()
+    {
+        assertQuery("SELECT count(*) FROM tpch.information_schema.tables WHERE table_name = 'orders'", "VALUES (9)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.tables WHERE table_name < 'orders'", "VALUES (30)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.tables WHERE table_name < 'orders'", "VALUES (30)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.columns WHERE table_name = 'orders'", "VALUES (81)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.columns WHERE table_name < 'orders'", "VALUES (267)");
+    }
+
+    @Test
+    public void testMixedPredicate()
+    {
+        assertQuery("SELECT * FROM tpch.information_schema.tables WHERE table_schema = 'sf1' and table_name = 'orders'", "VALUES ('tpch', 'sf1', 'orders', 'BASE TABLE')");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.columns WHERE table_schema = 'sf1' and table_name = 'orders'", "VALUES (9)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.tables WHERE table_schema > 'sf1' and table_name < 'orders'", "VALUES (24)");
+        assertQuery("SELECT count(*) FROM tpch.information_schema.columns WHERE table_schema > 'sf1' and table_name < 'orders'", "VALUES (224)");
```

Comment:
can you please also add some tests with `LIKE` operator

Reply:
Sure. There's one in `TestInformationSchemaConnector::testSchemaNamePredicate`.

I can add more but I cannot find any doc about `LIKE` in `prestosql.io` to learn about the expected behaviors. Any comments?

Reply:
`LIKE` is like simplified (wildcard) regular expression matching, where `_` matches to any single character and `%` matches to any (including no) characters.

`_l_` -> matches to `ola`, `ala` but not `ol` or `olka`
`_l%` -> matches to `ola`, `ala`, `ol` or `olka` but not `tola`

Reply:
Thanks, I've added more tests

Reply:
Thanks. I've added some tests